### PR TITLE
Feature/cart 142 property deletion

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/publicDev/admin.properties/style.scss [hunk 1]
```diff
@@ -131,14 +131,8 @@
     button {
```
Comment (dmitrij-borchuk) [inline dev/publicDev/admin.properties/style.scss]: Please move, this to common place and use css class

File: dev/publicDev/admin.properties/tpl.jade [hunk 2]
```diff
@@ -10,11 +10,16 @@
...
+          button.btn.btn-submit Search
         div(class="wrap-admin-properties")
           div(ng-repeat="item in vm.adminProperties | filter: vm.searchObj", class="pa-item")
             div(class="pa-img-wrap")
               // span(class="pa-img-bubbel") 45
               div(class="pa-blocked" ng-if="item.blocked") Blocked
               .logo(ng-style="{backgroundImage: 'url('+ item.logo + ')' }")
+            button.btn.btn-primary.btn-circle.delete(
+              type="button"
+              ng-hide="rule.editmode"
```
Comment (dmitrij-borchuk) [inline dev/publicDev/admin.properties/tpl.jade]: Rule?

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/publicDev/admin.properties/style.scss (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/publicDev/admin.properties/style.scss]: fixed

File: dev/publicDev/admin.properties/tpl.jade (hunk 2 above)
Comment (RomanFedytskyi) [reply dev/publicDev/admin.properties/tpl.jade]: fixed

**Review by dmitrij-borchuk — APPROVED** [review]

**Pull request merged**